src/clients/api/queries/getMarkets.ts

```diff
@@ -0,0 +1,30 @@
+import { restService } from 'utilities';
+import { VBEP_TOKENS } from 'constants/contracts';
+import { Market } from 'types';
+
+// interface IGetMarketsResponse {
+//   dailyVenus: string;
+//   markets: Market[];
+//   request: { addresses: string[] };
+//   venusRate: string;
+// }
+
```

Comment:
We can remove that.

Reply:
I think this would be nice to keep but I can remove it. It describes the shape of the data being returned but I didn't want to clutter eslint  with an unused variable name. I don't feel strongly about keeping it but I do think it provides useful information about what is going on here

Reply:
In that case, let's use it. If we want to go the extra mile, we can update the `restService` function so it can take a generic which types the returned data on success. This way you could pass this interface to it and have your response typed:
```
const response = restService<IGetMarketsResponse>({ api: '/governance/venus', method: 'GET' });
```

Reply:
I think this is a nice idea. I don't think it is worth digging more into this rest service at the moment because we will be replacing it. I think the comment works for now. I have no problem removing it because it is easy to find and recreate this information but I don't see the downside to keeping it. We have informative comments in other places as well.

I'm almost done with this update